### DISCUSSION TITLE
Pages of commands that are no longer maintained

### DISCUSSION THREAD

**Discussion by spageektti** [tldr-pages/tldr#12809] [Q&A] [answered]
I recently opened Issue #12808 about invalid links. 
I noticed that the repositories of many of these commands with not working links have been removed or archived, and their documentation is no longer available. 
What to do with such pages? Remove the page, or change the link from the non-available documentation to the archived repo?

**Comment by kbdharun** [comment, marked as answer]
Hi, It isn't required to remove a page when the link isn't working or becomes unavailable. While we suggest having more information links in pages for additional insights, it is still an optional part of the page.

When the documentation becomes unavailable we could link to the archived source repository if available or alternatively link to a snapshot in archival services like Internet Archive or if you can't find any relevant documentation or source for the tool then you can remove the more information link line.
